src/components/va-select/va-select.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  State,
+  h,
+} from '@stencil/core';
+import { getSlottedNodes } from '../../utils/utils';
+
+@Component({
+  tag: 'va-select',
+  styleUrl: 'va-select.css',
+  shadow: true,
+})
+export class VaSelect {
+  @Element() el!: any;
```

Comment:
TypeScript n00b question: What's this `!`? The closest I've found is the [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator), but this seems different. :thinking:

Reply:
I think it might be the non-null assertion operator, but now I'm not sure how this ended up here. I thought I copy-pasted it from [the docs](https://stenciljs.com/docs/host-element#element-decorator), but they don't have the `!`. Maybe they did at one point and have since been changed :thinking: 

Anyway, looking at the docs again, I don't think it is necessary here, so I can remove it.